client/state/selectors/README.md

```diff
@@ -9,5 +9,49 @@ When adding a new selector to this directory, make note of the following details
 
 - Each new selector exists in its own file, named with [kebab case](https://en.wikipedia.org/wiki/Kebab_case) (dash-delimited lowercase)
 - There should be no more than a single default exported function per selector file
+- The JSDoc header for the default exported function should contain an alias tag with the name of the function in it and that should match the camelCase conversion of the filename itself
```

Comment:
@aduth how do you feel about using `@selector` to indicate that a function is a selector. it would actually relax our rules for the search and could possibly allow us to build the `index.js` file automatically.